sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -628,3 +628,27 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 #IfNotRow2D list_options list_id Plan_of_Care_Type option_id intervention
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`) VALUES ('Plan_of_Care_Type', 'intervention', 'Intervention', '9', '0', '0', '', 'RQO', '', '0', '0', '1', '', '1');
 #EndIf
+
+#IfNotTable valueset_oid
+CREATE TABLE `valueset_oid` (
+  `nqf_code` varchar(255) NOT NULL DEFAULT '',
+  `code` varchar(255) NOT NULL DEFAULT '',
+  `code_system` varchar(255) NOT NULL DEFAULT '',
+  `code_type` varchar(255) DEFAULT NULL,
+  `valueset` varchar(255) NOT NULL DEFAULT '',
+  `description` varchar(255) DEFAULT NULL,
+  `valueset_name` varchar(500) DEFAULT NULL,
+  PRIMARY KEY (`nqf_code`,`code`,`valueset`)
+) ENGINE=InnoDB;
+#EndIf
+
+#IfNotRow code_types ct_key OID
+DROP TABLE IF EXISTS `temp_table_one`;
+CREATE TABLE `temp_table_one` (`id` int(11) NOT NULL DEFAULT '0',`seq` int(11) NOT NULL DEFAULT '0') ENGINE=InnoDB;
+INSERT INTO `temp_table_one` (`id`, `seq`) VALUES (
+  IF(((SELECT MAX(`ct_id` ) FROM `code_types`) >= 100), ((SELECT MAX(`ct_id` ) FROM `code_types`) + 1), 100),
```

Comment:
I don't see anything broken here, just wondering why we want the code_types ct_id and ct_seq to start at or be greater than 100.  Is that our default in databases.sql?

Reply:
right, basically just copied the stuff over from when we brought in the `valueset` table

Reply:
the 100 is from when these code_types were all hardcoded (just supported icd9/cpt4) which will note are single digits. When added more flexible support for code types, we decided to start them at 100 in upgrade scripts to ensure didn't break any instances that may of manually added other code types. pretty much just historical, but easier to just reuse these code blocks since we know they will work as expected :)